Dockerfile

```diff
@@ -14,6 +14,6 @@ CMD [ "uwsgi", "--ini", "uwsgi.ini" ]
 RUN groupadd uwsgi && useradd -g uwsgi uwsgi && mkdir -p $APP $APP/data && chown -R uwsgi $APP
 USER uwsgi
 # copy the whole webapp
-COPY ./uwsgi.ini upload_license.txt config.yaml ./
+COPY uwsgi.ini upload_license.txt ./
```

Comment:
Ok for the config.yaml, 
But also, we have a path on .env for the license, but this is build during container time? In shuch case it is not configurable and then I would hardcode it. Otherwise, I would pass it on the docker compose.